pom.xml

```diff
@@ -5,7 +5,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>uk.gov.service.notify</groupId>
     <artifactId>notifications-java-client</artifactId>
-    <version>3.4.0-RELEASE</version>
+    <version>3.4.1-TEST</version>
```

Comment:
I would assume this should be 
`3.5.0-RELEASE`
or
`3.5.0-SNAPSHOT`

Reply:
I am not sure what it should be - if you give me a hint, I will adjust the code accordingly.

Reply:
from this history of this file:
https://github.com/alphagov/notifications-java-client/commits/master/pom.xml
It's always checked in at a released version,
I would think bump it to 3.5.0-RELEASE

Reply:
Version bumped to 3.5.0-RELEASE